### DISCUSSION TITLE
Clear content

### DISCUSSION THREAD

**Discussion by kopitar** [spatie/laravel-ray#58] [Ideas]
It would be handy if I could clear any previous content (without CTRL + (Shift) K). Something like:

`ray()->clear();`

**Comment by freekmurze** [comment]
You can always see previous content by pressing the little back button on top of the window.

**Reply by kopitar** [reply]
What I meant was clearing the content of the currently active screen. So you get a fresh output each time.

Like if you run this 2 times:
```
ray('test 1');
ray('test 2');
```

You get:
```
test 1
test 2
test 1
test 2
```

What I meant was:
```
ray()->clear();
ray('test 1');
ray('test 2');
```

You get:
```
test 1
test 2
```

**Reply by freekmurze** [reply]
You can `newScreen` for this 👍
